src/scripts/check-definitions.py

```diff
@@ -33,18 +33,13 @@
 # For the native part, return the sorted definition array.
 def loadDefinitionFile(filename):
     result = []
-    try:
-        f = open(filename, 'r')
-    except:
-        sys.exit(0)
-    # if cmake was not used (because of skipnative or systems that do not use cmake), this script won't work.
-
-    for line in f:
-        theLine = line.rstrip("\r\n").strip()
-        if (len(theLine) > 0):
-            result.append(theLine)
-
-    f.close()
+
+    with open(filename, 'r') as f:
```

Comment:
I don't have enough context to be able to say if this is OK.  Can you comment on the differences here?  Also, I assume that the changes in this file are not related to EventPipe and could be in a separate PR if we so desired?

Reply:
This is similar to `using` in C#. (It still ensures the file gets closed out if an exception is thrown and it scopes the variable to the life of the handle.)

I did it head because I made the change to add of the eventing scripts (out of necessity) and saw this one too. I can split it out into another PR (possibly a Python refactor PR, which I would be good for maintainability if there is ever time for it)

Reply:
I did just realise there is an issue in that I removed the `sys.exit(0)`. Adding that back in